src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -100,7 +98,6 @@ public CommandResult execute(Model model) throws CommandException {
         }
 
         model.setCandidate(candidateToEdit, editedCandidate);
-        model.updateFilteredCandidateList(PREDICATE_SHOW_ALL_CANDIDATES);
```

Comment:
Thanks for making the change to fix the bug! 😄 After merging will try to do more extensive testing